ScnSoft-Education/LeetCodeTraining.Tests/3SumTest.cs

```diff
@@ -0,0 +1,46 @@
+﻿using LeetCodeTraining.ThreeSum;
+
+namespace LeetCodeTraining.Tests
+{
+    public class ThreeSumSolutionTest
+    {
+        [Theory]
+        [InlineData("-1,0,1,2,-1,-4", "[-1,-1,2],[-1,0,1]")]
```

Comment:
Please try to use MemberData attribute to pass input and expected data similar to:

```
        [MemberData(nameof(Data))]
        public void ThreeSumTest(int[] nums, IList<IList<int>> expectedResult)
        
         public static IEnumerable<object[]> Data =>
       new List<object[]>
       {
           new object[]
           {
               new int[] { -1, 0, 1, 2, -1, -4 },
               new List<IList<int>>()
               {
                   new List<int>() { -1, -1, 2 },
                   new List<int>() { -1, 0, 1 }
               }
           }...
         }
```